.github/workflows/publish.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: Publish
+"on":
+  push:
+    tags:
+      - '[1-3].*'
```

Comment:
Probably better to use `[1-9]` to future proof?

Reply:
🤔 I wonder... on one hand, less work two years from now.

On the other, if someone pushed a  8.1.0 tag (for scenario), I wonder if I would want this to job to get triggered.

Or maybe we can decide that it's kindof intended?